VersionControlUI/CMakeLists.txt

```diff
@@ -8,8 +8,10 @@ add_library(VersionControlUI SHARED
 	src/VersionControlUIException.h
 	src/versioncontrolui_api.h
 	src/VersionControlUIPlugin.h
+	src/DiffManager.h
 	src/VersionControlUIException.cpp
 	src/VersionControlUIPlugin.cpp
+	src/DiffManager.cpp
 	test/SimpleTest.cpp
 )
-envision_plugin(VersionControlUI InteractionBase)
+envision_plugin(VersionControlUI InteractionBase oomodel)
```

Comment:
So as a general rule a commit should be a more complete entity. Here you're adding a dependency but it is totally not obvious why we need this dependency. It should have been done in a commit that also introduces a use of something that's in oomodel.